sr_gui_controller_tuner/src/sr_gui_controller_tuner/controller_tuner.py

```diff
@@ -498,53 +537,56 @@ def refresh_controller_tree_(self, controller_type = "Motor Force"):
                         frame_buttons = QFrame()
                         layout_buttons = QHBoxLayout()
                         btn_plot = QPushButton("Plot")
-                        self.ctrl_widgets[ motor_name ]["btn_plot"] = btn_plot
-                        self.ctrl_widgets[ motor_name ]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_, motor_name, self.ctrl_widgets[ motor_name ]["btn_plot"]))
+                        self.ctrl_widgets[motor_name]["btn_plot"] = btn_plot
+                        self.ctrl_widgets[motor_name]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_,
+                                                                                  motor_name, self.ctrl_widgets[motor_name]["btn_plot"]))
                         layout_buttons.addWidget(btn_plot)
 
                         if self.controller_type in ["Position", "Muscle Position", "Mixed Position/Velocity"]:
                             #only adding Move button for position controllers
                             btn_move = QPushButton("Move")
-                            self.ctrl_widgets[ motor_name ]["btn_move"] = btn_move
-                            self.ctrl_widgets[ motor_name ]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_,motor_name, self.ctrl_widgets[ motor_name ]["btn_move"]))
+                            self.ctrl_widgets[motor_name]["btn_move"] = btn_move
+                            self.ctrl_widgets[motor_name]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_, motor_name,
+                                                                                      self.ctrl_widgets[motor_name]["btn_move"]))
                             layout_buttons.addWidget(btn_move)
                             frame_buttons.setLayout(layout_buttons)
 
-                        self._widget.tree_ctrl_settings.setItemWidget(  motor_item, 0, frame_buttons )
+                        self._widget.tree_ctrl_settings.setItemWidget(motor_item, 0, frame_buttons)
 
-                    for index_item,item in enumerate(ctrl_settings.headers):
+                    for index_item, item in enumerate(ctrl_settings.headers):
                         if item["type"] == "Bool":
                             check_box = QCheckBox()
 
-                            if parameter_values["sign"] == 1.0:
-                                check_box.setChecked(True)
-
-                            check_box.setToolTip("Check if you want a negative sign\n(if the motor is being driven\n the wrong way around).")
-                            self._widget.tree_ctrl_settings.setItemWidget(  motor_item, index_item, check_box )
+                            param_name = item["name"].lower()
+                            param_val = parameter_values[param_name]
+                            check_box.setChecked(param_val)
+                            if "sign" in param_name:
```

Comment:
@guihomework word "sign" is part of the such words as "design", "assign", etc. Do you think you could do more restrictive check such as "sign" == param_name or param_name.endswith("/sign") if you don't have time, we'll do our best to fix it. But it would be great if you provide expected parameter names to be verified. 
This remark is apply to some "in" statements in def check_prefix(self) for strings. If this is just check for special cases it is better to list them and make checks more solid. And of course it would be great if you provide possible values so in case if you do not have time we'll use it as reference for exact validation implementation.